src/transaction/send.ts

```diff
@@ -344,4 +344,9 @@ async function awaitTransactionSignatureConfirmation(
   done = true;
   log.debug("Returning status", status);
   return status;
+}
+
+export interface SentTransactionDetails {
```

Comment:
Let’s call this SendTransactionResult. Ties the naming closer to the return of the function call vs. details about the sent transaction. If we end up adding more information in the return that might not just be related to the transaction itself, the name will still fit.